packages/@okta/vuepress-site/docs/guides/device-context/main/nodeexpress/samplecode.md

```diff
@@ -0,0 +1 @@
+[Embedded Javascript SDK sample application](https://github.com/okta/okta-auth-js/tree/master/samples/generated/express-embedded-auth-with-sdk)
```

Comment:
What about the Java language?

Reply:
Not sure what you mean. There has never been a Java version of this article. Brian didn't finish it to my knowledge, or at least it never made it to a pull request

Reply:
I meant whether we also want to include a Java version of the document. Currently, the PR includes Asp.net and Node Express. Will having those two languages only meet expectations?

Reply:
@robertnicolo-okta We probably should ask Dan Marma, then add it as a new JIRA ticket? I'm not sure why it's not still an open ticket assigned to Brian Demers but I can't find it.